modules/cask-angular-sortable/sortable.js

```diff
@@ -47,17 +47,29 @@ function caskSortableDirective ($log) {
 
       angular.forEach(headers, function(th) {
         th = angular.element(th);
-        var a = th.attr('data-predicate-default');
-        if(angular.isDefined(a)) {
+        var a;
+
+        if (!$stateParams.sortBy) {
+          a = th.attr('data-predicate-default');
+        }
+
+        if ($stateParams.sortBy && th.attr('data-predicate') === $stateParams.sortBy) {
+          defaultPredicate = th;
+          defaultReverse = ($stateParams.reverse==='reverse');
+        } else if (angular.isDefined(a)) {
           defaultPredicate = th;
           defaultReverse = (a==='reverse');
         }
       });
 
-      if(!defaultPredicate) {
+      if (!defaultPredicate) {
         defaultPredicate = headers.eq(0);
       }
 
+      $stateParams.sortBy = defaultPredicate.attr('data-predicate');
```

Comment:
We don't need to do this I believe. ```$state.go(...)``` will update the `$stateParams` right? We should instead update this is in the object we pass in go.

Something like 

```
$state.go($state.$current.name, { 
  sortBy:  defaultPredicate.attr('data-predicate'),
  reverse: defaultReverse ? 'reverse' : '"
}, {
  notify: false
})
```